tools/cldr-code/src/main/java/org/unicode/cldr/util/CLDRFile.java

```diff
@@ -3623,8 +3736,14 @@ private String getLstrFallback(String codeType, String code) {
                 temp = temp2.get(0);
                 final Matcher matcher = CLEAN_DESCRIPTION.matcher(temp);
                 if (matcher.lookingAt()) {
-                    return matcher.group(1).trim();
+                    temp = matcher.group(1).trim();
                 }
+                // TODO: this code is not called anywhere where 'paths' is used at present.
+                // if (temp != null) {
+                //     if (paths != null) {
+                //         paths.add("//lstreg/"+codeType+"[@type=\"" + code + "\"]"); // pseudo-xpath
+                //     }
+                // }
```

Comment:
The TODO is vague -- would the commented-out code suffice supposing 'paths' were used? if so, just uncomment now? Otherwise, could throw an exception if paths isn't null?

I've been trying to avoid TODO comments (except in draft PR) unless they're accompanied by ticket numbers -- otherwise they tend to become wishful thinking that just clutters the code

Reply:
done